### DISCUSSION TITLE
Welcome to flutter_shortcuts Discussions!

### DISCUSSION THREAD

**Discussion by divshekhar** [divshekhar/flutter_shortcuts#9] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by baimamboukar** [comment]
Hii..! I'm <h3>Baimam John James </h3>(<a target="blank">https://twitter.com/baimamjj</a>)
from Cameroon ! 

How to create a "Unified Experience App" for all different platforms that flutter supports ? (web, android, ios, mac, windows, linux, etc...) ?
